src/index.jsx

```diff
@@ -38,6 +38,7 @@ import './legacy/vendorStyles';
 import { currentSettings } from './scripts/settings/userSettings';
 import taskButton from './scripts/taskbutton';
 import RootApp from './RootApp.tsx';
+import { name as WEB_NAME, version as WEB_VERSION } from '../package.json';
```

Comment:
Does webpack complain about this import? It does on the dashboard page so I was planning to change it at some point...

Reply:
There are two possible solutions for that warning:

1) Like apphost.js, don't use named export but just import the complete document and read from that
2) Add build defines for the relevant fields

I opted for option 2 and just pushed that